app/tests/daemon_script.py

```diff
@@ -0,0 +1,27 @@
+"""
+Test script for daemonizing
+"""
+
+from sys import argv
+from time import sleep
+
+from onedrive_client.utils import daemon
+
+
+def main():
+    """
+    Dummy function
+    """
+
+    while True:
+        sleep(5)
+
+
+# pylint: disable=invalid-name
+daemon = daemon.Daemon(app='testapp', pid=argv[2], action=main)
+if argv[1] == 'start':
```

Comment:
action = argv[1]
if action not in ACTIONS:
    print "invalid or excess argument"
    sys.exit(2)

\# Start action loop here
if action == 'start':
    daemon.start()
elif ....